src/utility.js

```diff
@@ -139,8 +139,10 @@ function isIterable(i) {
  * @returns true if e is an error
  */
 function isError(e) {
-  // Detect both Error and Firefox Exception type
-  return isType(e, 'error') || isType(e, 'exception');
+  // Detect Error, Firefox Exception type and error like object
+  return (isType(e, 'error')
+    || isType(e, 'exception')
+    || (typeof e === 'object' && e && e.name && e.message));
```

Comment:
I did not save the `typeName` because of this https://github.com/rollbar/rollbar.js/pull/915#issuecomment-763273211